appleseed-Installer/ProgressDialog.wxs

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--//**********************************************************************/
+// Copyright (c) 2021 appleseed.
+// All rights reserved.
+// 
+//**************************************************************************/
+// DESCRIPTION: Contains the Dll Entry stuff
+// AUTHOR: Omid Ghotbi (TAO) omid.ghotbi@gmail.com www.3dfine.com
+//***************************************************************************/-->
+<Wix xmlns="http://schemas.microsoft.com/wix/2006/wi">
+  <Fragment>
+    <UI Id="ProgressDialogUI">
+
+      <DialogRef Id="FinishedDialog"/>
+      <DialogRef Id="ErrorDialog"/>
+
+      <!--<Binary Id="headerPic" src="images/header.png"/>-->
+
+      <TextStyle Id="TahomaBig" FaceName="Tahoma" Size="18" Bold="yes" Red="255" Green="255" Blue="255" />
+
+      <Dialog Id="ProgressDialog" Width="555" Height="416" Title="Silly Software" Modeless="yes">
+
+        <Control Id="background" Type="Bitmap" Text="bgImage" Width="555" Height="416" X="0" Y="0" TabSkip="no" FixedSize="yes"/>
+
+        <!--<Control Id="header" Type="Bitmap" Text="headerImage" Height="50" Width="400" X="0" Y="0" TabSkip="no" />-->
+
+        <Control Id="installationText"  Type="Text" X="235" Y="110" Width="290" Height="40" Transparent="yes" Text="{\TahomaBig}appleseed Plugin Installation" />
+
+        <Control Id="waitText" X="230" Y="150" Width="290" Height="100" NoWrap="no" RightAligned="no" Transparent="yes" Type="Text" Text="{\TahomaNormal}Setup program will automatically do all necessarily steps and after it's finished you can use the plugin by start 3dsmax and go to the renderer options. Please wait while setup installs appleseed plugin on your computer. Click on the Cancel button to exit." />
```

Comment:
Just to clarify: Is this dialog showing _during_ appleseed installation, or just before?

Also, it seems that some of this text is being repeated from an earlier dialog.

If that's indeed the case, then what about simply this:

    The installation program is now installing appleseed on your computer.